mappings/net/minecraft/server/world/ServerEntityManager.mapping

```diff
@@ -0,0 +1,98 @@
+CLASS net/minecraft/class_5579 net/minecraft/server/world/ServerEntityManager
+	FIELD field_27260 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_27261 entityUuids Ljava/util/Set;
+	FIELD field_27262 entityLoader Lnet/minecraft/class_5576;
+	FIELD field_27263 entityDataAccess Lnet/minecraft/class_5571;
+	FIELD field_27264 trackedEntities Lnet/minecraft/class_5570;
+	FIELD field_27265 sectionedEntities Lnet/minecraft/class_5573;
+	FIELD field_27266 entityStore Lnet/minecraft/class_5577;
+	FIELD field_27267 statusByChunk Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+	FIELD field_27268 loadingStagesByChunk Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+	FIELD field_27269 tickingChunkSections Lit/unimi/dsi/fastutil/longs/LongSet;
+	FIELD field_27270 loadingQueue Ljava/util/Queue;
+	METHOD <init> (Ljava/lang/Class;Lnet/minecraft/class_5576;Lnet/minecraft/class_5571;)V
+		ARG 1 entityClass
+		ARG 2 entityLoader
+		ARG 3 entityDataAccess
+	METHOD method_31809 tick ()V
+	METHOD method_31810 ensureEntityChunkDataLoaded (J)V
+		ARG 1 chunkPos
+	METHOD method_31811 removeIfEmpty (JLnet/minecraft/class_5572;)V
+		ARG 1 sectionPos
+		ARG 3 entities
+	METHOD method_31812 tryWriteChunk (JLjava/util/function/Consumer;)Z
+		COMMENT Attempts to write the entity data for entities at the given {@link ChunkPos}, and runs the given
+		COMMENT action for each entity after saving.
+		COMMENT
+		COMMENT @return True if the entities were written, false if they were not.
+		ARG 1 chunkPos
+		ARG 3 postWriteCallback
+	METHOD method_31813 (Lnet/minecraft/class_4456;J)V
+		ARG 2 pos
+	METHOD method_31815 updateEntityChunkStatus (Lnet/minecraft/class_1923;Lnet/minecraft/class_3193$class_3194;)V
+		ARG 1 pos
+		ARG 2 levelType
+	METHOD method_31816 updateEntityChunkStatus (Lnet/minecraft/class_1923;Lnet/minecraft/class_5584;)V
+		ARG 1 pos
+		ARG 2 status
+	METHOD method_31818 addEntity (Lnet/minecraft/class_5568;)Z
+		ARG 1 entity
+	METHOD method_31820 addEntity (Lnet/minecraft/class_5568;Z)Z
+		ARG 1 entity
+	METHOD method_31826 dumpEntityDataToCsv (Ljava/io/Writer;)V
+		ARG 1 writer
+	METHOD method_31827 containsEntity (Ljava/util/UUID;)Z
+		ARG 1 entityUuid
+	METHOD method_31828 addEntitiesWithFlagSet (Ljava/util/stream/Stream;)V
```

Comment:
This naming `WithFlagSet` and `WithFlagCleared` is very vague, and probably not worth including unless using a proper name. As far as I can tell, this flag looks to relate to whether the entities are being loaded from a "legacy" world chunk that contains entities, or if it is being loaded from the new entity chunks.

Reply:
inherited from the previous PR and I agree, the flag doesn't actually do anything at the moment so it's difficult to name the methods surrounding it

Reply:
Resolved in a6fe16a9c